enhancements/etcd/disaster-recovery-with-ceo.md

```diff
@@ -0,0 +1,131 @@
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Current disaster recovery scripts were created before the introduction of `cluster-etcd-operator` (CEO). Cluster-etcd-operator (CEO) is an operator that handles the scaling of etcd during cluster bootstrap and regular operation and provisioning etcd dependencies such as TLS certificates. This enhancement allows modifications to the DR scripts such that disaster recovery makes use of the presence of CEO.
+
+
+## Motivation
+
+The immediate need for enhancement of disaster recovery scripts is to allow basic backup and restore operations to work, which are currently failing the automated tests (e2edisruptive tests). 
+
+In addition, the disaster recovery should allow the recovery of a single failed node, scaling up and scaling down (node drain).
+
+## Goals
+
+1.  If one master is lost, instructions for how to.
+    1. create a new master that joins the cluster
+    1. removal of the old master from the cluster
+1.  Changes to the etcd-quorum recovery steps
+1.  All nodes being shut off at the same time and restarted.
+1.  IP address change of a single member
+1.  IP address change of all members
+1.  debugging and detection when DNS information for all members is lost
+1.  debugging and detection when DNS information for one member is lost
+1.  Removal of a member from the etcd cluster
+1.  Recovery of a member with a bad data-dir.
+1.  Addition of a new member when there is significant etcd data.
+1.  Upgrade, downgrade, re-upgrade
+1.  Restoring to a previous state
+1.  Remove responsibility for these files from the MCO
+
+## Non-Goals
+
+*  Recover from expired certificates (not considered for 4.4)
+
+## Currently Supported Functionality 
+
+Currently the scripts available support the following functionality:
+### Backup ([usr-local-bin-etcd-snapshot-backup-sh.yaml](https://github.com/openshift/machine-config-operator/blob/3e10747951dd1c3c7a9f131d4c5af7805f19a164/templates/master/00-master/_base/files/usr-local-bin-etcd-snapshot-backup-sh.yaml))
+Takes a snapshot of cluster’s etcd data along with static-pod-resources at the time of the backup. 
+
+### Restore ([usr-local-bin-etcd-snapshot-restore-sh.yaml](https://github.com/openshift/machine-config-operator/blob/3e10747951dd1c3c7a9f131d4c5af7805f19a164/templates/master/00-master/_base/files/usr-local-bin-etcd-snapshot-restore-sh.yaml))
+Restores the etcd data from a backup snapshot. It also restores the static pod resources while deleting all the newer revisions.
+
+### Remove an etcd member ([usr-local-bin-etcd-member-remove-sh.yaml](https://github.com/openshift/machine-config-operator/blob/3e10747951dd1c3c7a9f131d4c5af7805f19a164/templates/master/00-master/_base/files/usr-local-bin-etcd-member-remove-sh.yaml))
+This script removes a member from the etcd cluster membership. This is useful when trying to replace a failed master host.
```

Comment:
why is this better than `oc -n openshift-etcd rsh pod/working-pod -- ectdctl member remove bad-member`?